kernel/tests/read.rs

```diff
@@ -360,16 +364,6 @@ async fn stats() -> Result<(), Box<dyn std::error::Error>> {
     Ok(())
 }
 
-macro_rules! sort_lines {
-    ($lines: expr) => {{
-        // sort except for header + footer
-        let num_lines = $lines.len();
-        if num_lines > 3 {
-            $lines.as_mut_slice()[2..num_lines - 1].sort_unstable()
-        }
-    }};
-}
-
 // NB: expected_lines_sorted MUST be pre-sorted (via sort_lines!())
 macro_rules! assert_batches_sorted_eq {
```

Comment:
wasn't this also moved to common? why did it stay here?

Reply:
I have a separate macro in the new code that does the sorting inline.. probably could clean this up/unify etc..